lib/modules/platform/bitbucket/utils.ts

```diff
@@ -19,6 +19,7 @@ export function repoInfoTransformer(repoInfoBody: RepoInfoBody): RepoInfo {
     mergeMethod: 'merge',
     has_issues: repoInfoBody.has_issues,
     uuid: repoInfoBody.uuid,
+    isPrivate: repoInfoBody.is_private,
```

Comment:
```suggestion
    is_private: repoInfoBody.is_private,
```
can we use existing naming? otherwise it's confusing to have different namings